features/step_definitions/cli_steps.ts

```diff
@@ -26,14 +27,14 @@ When(
   async function (unparsedArgs) {
     // Use user's preferred shell to split args.
     const { stdout } = await execAsync(
-      `node -p "JSON.stringify(process.argv)" -- ${unparsedArgs}`
+      `node -p "JSON.stringify(process.argv)" -- ${unparsedArgs}`,
```

Comment:
prettier has changed the default settings to always place trailing commas when it's newline, didn't see any prettier config file to update